report/id.go

```diff
@@ -1,8 +1,13 @@
 package report
 
 import (
+	"hash"
+	"hash/fnv"
 	"net"
 	"strings"
+	"sync"
+
+	cache "github.com/hashicorp/golang-lru"
```

Comment:
Alvaro is already using a different LRU cache in one of the probe packages. Can we settle on one dependency? And can we not rename imports unless there's a conflict?